packages/components/src/Disclosure/src/Disclosure.tsx

```diff
@@ -0,0 +1,97 @@
+import { useStyledSystem, type StyledComponentProps } from "@hopper-ui/styled-system";
+import { forwardRef, useState, type ForwardedRef } from "react";
+import { composeRenderProps, Disclosure as RACDisclosure, useContextProps, type DisclosureProps as RACDisclosureProps } from "react-aria-components";
+
+import { composeClassnameRenderProps, cssModule, SlotProvider } from "../../utils/index.ts";
+
+import { DisclosureContext } from "./DisclosureContext.ts";
+import { DisclosureHeaderContext } from "./DisclosureHeaderContext.ts";
+import { DisclosurePanelContext } from "./DisclosurePanelContext.ts";
+import { InternalDisclosureContext } from "./InternalDisclosureContext.ts";
+
+import styles from "./Disclosure.module.css";
+
+export const GlobalDisclosureCssSelector = "hop-Disclosure";
+
+export type DisclosureVariant = "standalone" | "inline";
```

Comment:
why create a type for this? it's only used in the interface

Reply:
I figured in case users ever need to type our props values, we'd have them available.

Reply:
at the moment, the only thing this does is "obfuscate" the type in the documentation, so it's not a pattern we should use unless it's necessary